includes/alerts/transport.canopsis.php

```diff
@@ -0,0 +1,64 @@
+require_once "/usr/share/php/PhpAmqpLib/autoload.php";
```

Comment:
It seems that the amqplib-php software is under GPL 
https://github.com/php-amqplib/php-amqplib/blob/master/LICENSE